cinema/models.py

```diff
@@ -35,12 +39,22 @@ def full_name(self):
         return f"{self.first_name} {self.last_name}"
 
 
+def create_custom_path(instance: object, filename: str) -> str:
```

Comment:
instance
Please provide the correct type, you can specify more better type

Reply:
how?
Movie class is not defined yet

Reply:
you can use "" 

`def create_custom_path(instance: "Movie", filename: str) -> str:`